test-abi.sh

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env bash
+#
+# Easily run the ABI tests for the entire repo or a subset
+#
+
+here=$(dirname "$0")
+set -x
+exec ${here}/cargo nightly test --lib -- test_abi_
```

Comment:
I wonder why this slipped through the shellcheck on this pr's ci build... #13039

Reply:
well, `shellcheck` wasn't run at all. maybe there is a bug when dynamic buildpipeline.... I'll look at this later.

Reply:
https://buildkite.com/solana-labs/solana/builds/33226